app/views/idv/address/new.html.erb

```diff
@@ -9,14 +9,14 @@
 
 <% title t('titles.doc_auth.address') %>
 
-<% if @presenter.puerto_rico_address? %>
-  <%= render AlertComponent.new(
-        type: :info,
-        class: 'margin-bottom-4',
-      ) do %>
-    <%= t('doc_auth.info.address_guidance_puerto_rico_html') %>
-  <% end %>
-<% end %>
+<%= render AlertComponent.new(
+      type: :info,
+      id: 'puerto-rico-extra-text',
+      class: ['margin-bottom-4', 'display-none'],
```

Comment:
Could this also use `@presenter.hint_class`?

Reply:
Yup. It now does.